bcda/models/models.go

```diff
@@ -137,6 +135,26 @@ func (job *Job) GetEnqueJobs(t string) (enqueJobs []*que.Job, err error) {
 	return enqueJobs, nil
 }
 
+func GetMaxBeneCount(requestType string) int {
+	var envVar string
+	var defaultVal int
+
+	switch requestType {
+	case "ExplanationOfBenefit":
+		envVar = "BCDA_FHIR_MAX_RECORDS_EOB"
+		defaultVal = constants.BCDA_FHIR_MAX_RECORDS_EOB_DEFAULT
+	case "Patient":
+		envVar = "BCDA_FHIR_MAX_RECORDS_PATIENT"
+		defaultVal = constants.BCDA_FHIR_MAX_RECORDS_PATIENT_DEFAULT
+	case "Coverage":
+		envVar = "BCDA_FHIR_MAX_RECORDS_COVERAGE"
+		defaultVal = constants.BCDA_FHIR_MAX_RECORDS_COVERAGE_DEFAULT
+	}
```

Comment:
Do we want to add a `default` case to throw an error if it's an unexpected type? It looks like it would return 0 because `defaultVal` would automatically be 0 if it doesn't get set, which might be fine too. Curious what you think.

Reply:
Yep, thought about that default case. Didn't see the need since the calling function to this one (bulkRequest) already does the checking. And I believe its the only place where this function is being called, but maybe add it just in case for the future?? If GetEnqueJobs is later called by something else then yeah we should probably add that default case.